packages/loader/container-loader/src/container.ts

```diff
@@ -133,6 +134,7 @@ export interface IContainerLoadOptions {
      * Loads the Container in paused state if true, unpaused otherwise.
      */
     loadMode?: IContainerLoadMode;
+    loggerOverride?: ITelemetryBaseLogger;
```

Comment:
I am not sure "override" is a best name to use here. It's somewhat implying coupling with default functionality. Let's say some day this becomes the only way to set logger. The name itself would imply incorrect behavior.

Reply:
I used it because there's another header called ``clientDetailsOverride` in `IContainerLoadOptions` and `IContainerConfig`. How about `baseLogger`?